### ISSUE TITLE
Error when trying to start 23.1 or 23.2 

### ISSUE THREAD

**Issue by MacqGit** [Alfresco/alfresco-docker-installer#206]
Configuration:
? Which ACS version do you want to use? 23.2
? How may GB RAM are available for Alfresco (16 is minimum required)? 16
? Do you want to use HTTPs for Web Proxy? No
? What is the name of your server? localhost
? Choose the password for your admin user admin
? What HTTP port do you want to use (all the services are using the same port)? 80
? Do you want to use FTP (port 2121)? Yes
? Do you want to use MariaDB instead of PostgreSQL? No
? Are you using different languages (this is the most common scenario)? Yes
? Do you want to search in the content of the documents? Yes
? Would you like to use Shared Secret or HTTPs for Alfresco-SOLR communication? secret
? Do you want to use the Events service (ActiveMQ)? Yes
? Do you want to use credentials for Events service (ActiveMQ)? No
? Do you want to create an internal SMTP server? Yes
? Do you want to create an internal LDAP server? No
? Select the addons to be installed: Google Docs 3.1.0, JavaScript Console 0.7, Order of the Bee Support Tools 1.2.2.0, Share 
Site Creators 0.0.8, Share Site Space Templates 1.1.4-SNAPSHOT, Simple OCR 2.3.1 (for Alfresco 6.x), Alfresco OCR Transformer 
1.0.0 (for Alfresco 7+), ESign Cert 1.8.4, Edit with LibreOffice in Alfresco Share 0.3.0, Alfresco PDF Toolkit 1.4.4
? Do you want Docker to manage volume storage (recommended when dealing with permission issues)? Yes
? Do you want to use a start script? Yes
? Do you want to get the script to create host volumes? Yes

Clean environment via:
docker volume prune --filter all=1
docker container prune

Build & Start:
docker compose up --build --force-recreate

Everything seems to run fine until:
alfresco-1            | 27-Sep-2024 13:35:14.187 INFO [main] org.apache.catalina.startup.Catalina.start Server startup in [24043] milliseconds
alfresco-1            | 27-Sep-2024 13:35:36.898 INFO [QuartzScheduler_QuartzSchedulerThread] org.apache.catalina.loader.WebappClassLoaderBase.checkStateForResourceLoading Illegal access: this web application instance has been stopped already. Could not load [org.quartz.DisallowConcurrentExecution]. The following stack trace is thrown for debugging purposes as well as to attempt to terminate the thread which caused the illegal access.
alfresco-1            | 	java.lang.IllegalStateException: Illegal access: this web application instance has been stopped already. Could not load [org.quartz.DisallowConcurrentExecution]
[debug.log](https://github.com/user-attachments/files/17165011/debug.log)
. The following stack trace is thrown for debugging purposes as well as to attempt to terminate the thread which caused the illegal access.

Full log in attachement:
[debug.log](https://github.com/user-attachments/files/17165021/debug.log)

**Comment by aborroy** [comment]
Not all the addons are compatible with 23.2. I guess we'd need to add a control to verify the compatibility of them.

**Comment by MacqGit** [comment]
@aborroy Hello Angel, thanks for the feedback. Over the week-end, I realized that this might indeed be a compatibility issue with some of the extension modules.
I assume that what was reported in the logs was that the ACS didn't start correctly and eventually prevented Solr to connect to it.
Sorry for the burden but I'm rather familiar with Python and not that much with Java... :-)
However, just as a suggestion, wouldn't it be nice to put a little note about that in the docs ?
I spent quite some time testing all sort of configurations without realizing that the root cause actually laid in the misconfiguration of the generator...
Anyway, many thanks for the great job !!
